### ISSUE TITLE
Field prefix Underscore ignored when generating with method

### ISSUE THREAD

**Issue by bbaumgartner** [helospark/SparkBuilderGenerator#53]
Hi

This is an awesome plugin, thank you very much. We use Underscore "_" as field prefix. When generating a builder the prefix is not removed for the with method name. Example:

```
public class Test
{
   private final String _field;

   public Test (String field)
   {
      super();
      _field = field;
   }

   public String getField ()
   {
      return _field;
   }
}
```

results in

```
public static final class Builder
   {
      private String _field;

      private Builder ()
      {
      }

      public Builder with_field (String _field)
      {
         this._field = _field;
         return this;
      }

      public Test build ()
      {
         return new Test(this);
      }
   }
```

but should:

```
     public Builder withField (String field)
      {
         this._field = field;
         return this;
      }
```

It works for other prefixes or when nameing the field _Field instead of _field.

Problem is located in FieldNameToBuilderFieldNameConverter  line 40:
```
.filter(prefix -> isCharacterAfterPrefixCapital(fieldName, prefix))
```

Thank you

**Comment by helospark** [comment]
Thank you for raising this detailed issue, I will release the fix for this within a few days.

**Comment by helospark** [comment]
Released version 0.0.22 with the fix.

**Comment by bbaumgartner** [comment]
Thank you very much.
